tests/mocks.py

```diff
@@ -478,6 +477,35 @@ def transaction(
         )
         return transaction
 
+    def comment(self, content, author=None):
+        """Return a Phabricator Comment object.
+
+        Args:
+            content: The raw (Remarkup) comment contents.
+            author: Optional Phabricator User that authored the comment. If None
+                then a new User will be generated.
+        """
+        phid = self._new_phid("XCMT-")
+        author = self.user() if author is None else author
+        comment = {
+            "id": self._new_id(self._comments),
+            "phid": phid,
+            "version": 1,
+            "authorPHID": author["phid"],
+            "dateCreated": 1570502242,
+            "dateModified": 1570502242,
+            "removed": False,
+            "content": {"raw": content},
+        }
+
+        self._comments.append(comment)
+        # NOTE: Comments are an internal type. They cannot be introspected. We don't
+        # know the value of a number of the fields that other API objects have,
+        # such as 'type' and 'uri'.
```

Comment:
This comment placement confuses me, what is it supposed to apply to? I'm not exactly sure what this is trying to say or what makes comments unique compared to all the other internal representations we have?

Reply:
I removed the comment and the PHID registration since calling `phid.lookup` on a comment doesn't return any result with a real Phabricator instance.